testsuite/features/secondary/srv_manage_activationkey.feature

```diff
@@ -0,0 +1,99 @@
+    And I follow "SUSE Test Key i586"
+    And I select "Test-Channel-i586" from "selectedBaseChannel"
+    And I click on "Update Activation Key"
+    Then I should see a "Activation key SUSE Test Key i586 has been modified." text
+
+  Scenario: Create an activation key with a channel and a package list for i586
+    Given I am on the Systems page
+    When I follow the left menu "Systems > Activation Keys"
+    And I follow "Create Key"
+    And I enter "SUSE Test PKG Key i586" as "description"
+    And I enter "SUSE-TEST-2-i586" as "key"
+    And I enter "20" as "usageLimit"
+    And I select "Test-Channel-i586" from "selectedBaseChannel"
+    And I click on "Create Activation Key"
+    And I follow "Packages"
+    And I enter "sed" as "packages"
+    And I click on "Update Activation Key"
+    Then I should see a "Activation key SUSE Test PKG Key i586 has been modified." text
+    And I should see a "Details" link
+    And I should see a "Packages" link
+    And I should see a "Configuration" link in the content area
+    And I should see a "Groups" link
+    And I should see a "Activated Systems" link
+
+  Scenario: Create an activation key with a channel and a package list for x86_64
+    Given I am on the Systems page
+    When I follow the left menu "Systems > Activation Keys"
+    And I follow "Create Key"
+    And I enter "SUSE Test PKG Key x86_64" as "description"
+    And I enter "SUSE-TEST-x86_64" as "key"
+    And I enter "20" as "usageLimit"
+    And I select "Test-Channel-x86_64" from "selectedBaseChannel"
+    And I click on "Create Activation Key"
+    And I follow "Packages"
+    And I enter "sed" as "packages"
+    And I click on "Update Activation Key"
+    Then I should see a "Activation key SUSE Test PKG Key x86_64 has been modified." text
+    And I should see a "Details" link
+    And I should see a "Packages" link
+    And I should see a "Configuration" link in the content area
+    And I should see a "Groups" link
+    And I should see a "Activated Systems" link
+
+  Scenario: Create an activation key with a channel and a package list for x86_64
+    Given I am on the Systems page
+    When I follow the left menu "Systems > Activation Keys"
+    And I follow "Create Key"
+    And I enter "SUSE Test PKG Key x86_64" as "description"
+    And I enter "SUSE-TEST-x86_64" as "key"
+    And I enter "20" as "usageLimit"
+    And I select "Test-Channel-x86_64" from "selectedBaseChannel"
+    And I click on "Create Activation Key"
+    And I follow "Packages"
+    And I enter "sed" as "packages"
+    And I click on "Update Activation Key"
+    Then I should see a "Activation key SUSE Test PKG Key x86_64 has been modified." text
+    And I should see a "Details" link
+    And I should see a "Packages" link
+    And I should see a "Configuration" link in the content area
+    And I should see a "Groups" link
+    And I should see a "Activated Systems" link
+
+  Scenario: Cleanup: delete test keys
+# TBD
```

Comment:
There is any issue about adding the cleanup of the above created keys in this PR?

Reply:
I just still need to write it.

It's for idempotency.

Reply:
Written as 3 more scenarios